internal/services/loganalytics/log_analytics_cluster_customer_managed_key_resource.go

```diff
@@ -19,9 +19,9 @@ import (
 
 func resourceLogAnalyticsClusterCustomerManagedKey() *pluginsdk.Resource {
 	return &pluginsdk.Resource{
-		Create: resourceLogAnalyticsClusterCustomerManagedKeyCreate,
+		Create: resourceLogAnalyticsClusterCustomerManagedKeyCreateOrUpdate,
 		Read:   resourceLogAnalyticsClusterCustomerManagedKeyRead,
-		Update: resourceLogAnalyticsClusterCustomerManagedKeyUpdate,
+		Update: resourceLogAnalyticsClusterCustomerManagedKeyCreateOrUpdate,
```

Comment:
we should not be combining creating and update functions as it breaks lifecycle - please keep this seperate